src/index.html

```diff
@@ -4,9 +4,334 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <div class="header__logo">
+        <img src="./images/logo.png" alt="Logo_Moyo">
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#pcs"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#laptops">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__content">
+      <section class="page__catalog catalog">
+        <div class="product-card catalog__item" data-qa="card">
```

Comment:
```suggestion
        <article class="product-card catalog__item" data-qa="card">
```
Not really sure if it would be better